core/drivers/crypto/stm32/stm32_cryp.c

```diff
@@ -84,6 +85,7 @@
 
 #define CRYP_TIMEOUT_US			1000000U
 #define TIMEOUT_US_1MS			1000U
+#define CRYP_RESET_DELAY		U(2)
```

Comment:
prefer explicitly state that the value is a microsecond time ref, 
for example s/CRYPT_RESET_DELAY/CRYPT_RESET_DELAY_US/g